libs/blockscout-service-launcher/src/launcher/span_builder.rs

```diff
@@ -2,21 +2,48 @@ use actix_web::{
     body::MessageBody,
     dev::{ServiceRequest, ServiceResponse},
     http::StatusCode,
-    Error, ResponseError,
+    Error, HttpMessage, ResponseError,
 };
 use once_cell::sync::Lazy;
-use std::{collections::HashMap, sync::Mutex, time::Instant};
+use std::{
+    collections::{HashMap, HashSet},
+    sync::{Mutex, RwLock},
+    time::Instant,
+};
 use tracing::{Id, Span};
 use tracing_actix_web::root_span;
 
 static REQUEST_TIMINGS: Lazy<Mutex<HashMap<Id, Instant>>> =
     Lazy::new(|| Mutex::new(HashMap::new()));
 
+static SKIP_HTTP_TRACE_PATHS: Lazy<RwLock<HashSet<String>>> =
+    Lazy::new(|| RwLock::new(HashSet::new()));
+
 #[derive(Default)]
 pub struct CompactRootSpanBuilder;
 
+impl CompactRootSpanBuilder {
+    pub fn init_skip_http_trace_paths<S: Into<String>>(
```

Comment:
why write this function inside CompactRootSpanBuilder? 👀

Reply:
Mostly to indicate that it is `CompactRootSpanBuilder` that uses those paths to be skipped